integration/forms/src/app/app.component.html

```diff
@@ -1,3 +1,3 @@
 <h1>Forms app</h1>
-<app-template-forms></app-template-forms>
-<app-reactive-forms></app-reactive-forms>
\ No newline at end of file
+<app-template-forms/>
```

Comment:
Don't forget the space:
```suggestion
<app-template-forms />
```